libs/tools/barista/src/transform.ts

```diff
@@ -300,3 +342,17 @@ export function internalContentTransformerFactory(
     return transformed;
   };
 }
+
+// Checks whether a URL is relative
```

Comment:
Method comments should be JsDoc. Because JsDoc is part of the function declaration and therefore it can be extracted for type information.

```suggestion
/** Checks whether a URL is relative */
```